content/influxdb/cloud-dedicated/primers/api/v1/_index.md

```diff
@@ -0,0 +1,275 @@
+
+
+### Authenticate with a username and password scheme
+
+With the InfluxDB v1 API, you can use the InfluxDB 1.x convention of username and password to authenticate database reads and writes by passing a [Database Token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token) as the `password` credential.
+When authenticating requests to the v1 API `/write` and `/query` endpoints, InfluxDB Cloud checks that `password` is an authorized Database Token and ignores `username`.
+
+Use one of the following authentication schemes with clients that support Basic authentication or query parameters (that don't support the `Authorization: Token` scheme):
+
+- [Basic authentication](#basic-authentication)
+- [Query string authentication](#query-string-authentication)
+
+#### Basic authentication
+
+Use the `Authorization` header with the `Basic` scheme to authenticate v1 API `/write` and `/query` requests.
+When authenticating requests, InfluxDB Cloud checks that `password` is an authorized Database Token and ignores `username`.
+
+##### Syntax
+
+```http
+Authorization: Basic <base64-encoded [USERNAME]:DATABASE_TOKEN>
+```
+
+Replace the following:
+
+- **`[USERNAME]`**: any value or leave empty. InfluxDB Cloud Dedicated ignores this part of the credentials.
+- **`DATABASE_TOKEN`**: a [Database Token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token).
+- Encode the `[USERNAME]:DATABASE_TOKEN` credentials using base64 encoding, and then append the encoded string to the `Authorization: Basic` header.
+
+{{% api/v1-compat/basic-auth-syntax %}}
+
+##### Example
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[curl](#curl)
+[Node.js](#nodejs)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+With an empty username:
+
+```sh
+{{% get-shared-text "api/cloud-dedicated/basic-auth.sh" %}}
+```
+{{% /code-tab-content %}}
+
+{{% code-tab-content %}}
+
+With an empty username:
+
+```js
+{{% get-shared-text "api/cloud-dedicated/basic-auth.js" %}}
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+Replace the following:
+
+- *`DATABASE_TOKEN`*: a [Database Token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token) with sufficient permissions to the database.
+
+#### Query string authentication
+
+Use the `p` query parameter to authenticate `/write` and `/query` requests.
+When authenticating requests, InfluxDB Cloud checks that `p` is an authorized Database Token and ignores the `u` parameter.
+
+{{% warn %}}
+##### Consider when using query string parameters
+
+- URL-encode query parameters that may contain whitespace or other special characters.
+- Be aware of the [risks](https://owasp.org/www-community/vulnerabilities/Information_exposure_through_query_strings_in_url) when exposing sensitive data through URLs.
+{{% /warn %}}
+
+##### Syntax
+
+```sh
+https://cloud2.influxdata.com/query/?[u=any]&p=DATABASE_TOKEN
+https://cloud2.influxdata.com/write/?[u=any]&p=DATABASE_TOKEN
+```
```

Comment:
Nothing to change here, just a note–We're going to need to update the URL functionality to let users set custom URLs in the cloud-dedicated docs, just like they do with OSS.